integration-cli/docker_cli_nat_test.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 func TestNetworkNat(t *testing.T) {
+	testRequires(t, NativeExecDriver)
 	testRequires(t, SameHostDaemon)
```

Comment:
i found a few more instances where defer was before lol so im going to fix those too